src/components/calcite-pick-list-item/calcite-pick-list-item.tsx

```diff
@@ -224,7 +221,7 @@ export class CalcitePickListItem {
       >
         {this.renderIcon()}
```

Comment:
Seems like this should be inside the `label` element with a role of `checkbox` no? The icon is a replacement for the input checkbox and clicking the label toggles it.

Reply:
For example: `<label><input /></label>` => `<label><icon /></label>`

Reply:
I think that's a good call - will look into modifying. @asangma this will require styling changes, I'll ping you once I'm done so you can review.